src/gameplay.js

```diff
@@ -83,14 +86,18 @@ const Gameplay = () => {
         return false;
     }
     const checkAppleCollision = newSnake => {
-        if (newSnake[0][0] === apple[0] && newSnake[0][1] === apple[1]) {
-            let newApple = createApple();
-            while (checkCollision(newApple, newSnake)) {
-                newApple = createApple();
+        for (let i = 0; i <= TONGUE_CONSTANTS[settings["tongueLength"]]; i++) {
```

Comment:
To improve readability, I would recommend adding a constant variable, something like
`tongueLen =  TONGUE_CONSTANTS[settings["tongueLength"]]`, 
and use this constant for the loop inside checkAppleCollision()